spec/jobs/skill_fetcher_job_spec.rb

```diff
@@ -1,5 +1,62 @@
 require 'rails_helper'
 
 RSpec.describe SkillFetcherJob, type: :job do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe '#fetch_org_info' do
+    let(:make_request) {
+      ->{
+        VCR.use_cassette('github_organization') do
+          GithubClient.new.fetch_org_info('openlexington')
+        end
+      }
+    }
+
+    it 'returns a resource' do
+      expect(make_request.call).to be_a(Sawyer::Resource)
+    end
+
+    it 'contains openlexington as the login' do
+      expect(make_request.call.login).to eq('openlexington')
+    end
+  end
+
+  describe '#fetch_org_member_list' do
+    let(:make_request) {
+      ->{
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Use the `lambda` method for multi-line lambdas.